cmake/extensions.cmake

```diff
@@ -1733,9 +1733,9 @@ Please provide one of following: APPLICATION_ROOT")
       if(NOT IS_ABSOLUTE ${path})
         message(FATAL_ERROR
 "Relative path encountered in scoped variable: ${ARGV1}, value=${path}\n \
-Please adjust any `set(${ARGV1} ${path}) or `list(APPEND ${ARGV1} ${path})\n \
+Please adjust any `set(${ARGV1} ${path})` or `list(APPEND ${ARGV1} ${path})`\n \
 to absolute path using `\${CMAKE_CURRENT_SOURCE_DIR}/${path}` or similar. \n \
-Relative paths are only allowed with `-D${ARGV1}=<path>")
+Relative paths are only allowed with `-D${ARGV1}=<path>`")
```

Comment:
Should be part of previous commit

Reply:
Of course.
Had forgotten that I did two commits in this PR when doing the amend.

Fixed.